exonum/src/node/state.rs

```diff
@@ -757,51 +737,23 @@ impl State {
         self.queued.push(msg);
     }
 
-    /// Returns non-committed transactions.
-    pub fn transactions(&self) -> &TxPool {
-        &self.transactions
-    }
 
-    /// Adds a transaction to the pool and returns list of proposes that don't contain unknown
-    /// transactions now.
+    /// Handles a transaction validated event and returns list of proposes
+    /// that don't contain unknown transactions now.
```

Comment:
Technically, this function doesn't handle any event, right? Documentation (and probably naming) should be "fixed".

Reply:
This method is called after transaction validation, how should i rename it (change documentation)?

Reply:
I think that docs for this method need to be updated/rewritten